build/packages.targets

```diff
@@ -34,7 +35,7 @@
         <PackageReference Update="Microsoft.CodeAnalysis.BannedApiAnalyzers" Version="3.3.2" />
         <PackageReference Update="Microsoft.CodeAnalysis.FxCopAnalyzers" Version="2.9.8" />
         <PackageReference Update="Microsoft.DataAI.NuGetRecommender.Contracts" Version="2.1.0" />
-        <PackageReference Update="Microsoft.Extensions.CommandLineUtils" Version="1.0.1" />
+        <PackageReference Update="Microsoft.Extensions.CommandLineUtils.Sources" Version="$($(MicrosoftExtensionsCommandLineUtilsSourcesPackageVersion))" />
```

Comment:
This has a double `$(` and that's failing the build.

Reply:
Fixed, thanks.